### ISSUE TITLE
HUL- 2 map Urban Typologies 1:1000

### ISSUE THREAD

**Issue by alxpapadaki** [FruitsofThought/ngambomap#13]
Urban Typologies

Produced in photoshop/illustrator.

Amélie is on that.

**Comment by batje** [comment]
Sorry, the QGIS map _already_ has this map. It may be ugly, but it works, and it gets improved all the time. See #20 and #2 and #4 
You can make really pretty maps in QGIS, that is what it is made for. And the _huge_ advantage is that you  only style once and while the data is updated, your map becomes better and better.

**Comment by alxpapadaki** [comment]
Yes true. But we want a different presentation for that. Maybe we should close it. we don't have to do something about it anymore. If the info is correct then we are fine.

**Comment by batje** [comment]
What 'Different Presentation' do you want?

**Comment by alxpapadaki** [comment]
for example: 
https://gr.pinterest.com/source/emilygarfield.com/
https://datavisualization.ch/showcases/mapping-istanbul-by-project-projects/
https://gr.pinterest.com/thefabric/02-maps-mapping/

**Comment by batje** [comment]
To do watercolor style in QGIS you can use
http://anitagraser.com/2015/01/04/how-to-watercolor-pastel-style-in-qgis/
(that is a rather simple process)

If you want to do it all the way, this is how Stamen builts up that effect
http://content.stamen.com/watercolor_process

Which ends up looking like
http://maps.stamen.com/watercolor/#15/-6.1642/39.1971

Can you pick 1 'Different Presentation' so that we can see how close we can match that in QGIS?
